.gitpod.yml

```
@@ -40,6 +40,7 @@ tasks:
       mongo --eval "db.fsyncLock(); db.fsyncUnlock()"
     command: >
       npm run ensure-env &&
+      npm run build:curriculum --if-present &&
```

```suggestion
      npm run build:curriculum &&
```
We want it to fail if `build:curriculum` isn't there, don't we?

---

I am not sure....
What does the flag do?

---

It's documented [here](https://docs.npmjs.com/cli/v6/commands/npm-run-script), but it means it will fail quietly.  I think we want to know if a script is missing, so I'd rather we saw an error.

---

As an example, when I broke the GitPod integration, it was pretty obvious.  So, it wasn't *good* that I broke it, but at least it was clear what had gone wrong!